aioftp/client.py

```diff
@@ -687,7 +764,7 @@ def __aiter__(cls):
                         name, info = cls.parse_line(line)
                     except KeyboardInterrupt:
                         raise
-                    except:  # noqa
+                    except Exception as e:  # noqa
```

Comment:
`# noqa` should be removed in this case. It was old bad times when I don't know about difference between `Exception` and `BaseException`. :smile: